### DISCUSSION TITLE
Why is my code coverage report is not showing the Increase and decrease

### DISCUSSION THREAD

**Discussion by prabhasjoshi** [ArtiomTr/jest-coverage-report-action#197] [Q&A] [answered]
Currently as per this diagram -

https://github.com/ArtiomTr/jest-coverage-report-action/blob/master/img/Rejected-PR-screenshot.png

- if i integrate with this via github action - it shows the increase and decrease in the code coverage.

But when I integrate with it - it's not showing it to me.

i'm using the latest version and this is what my yml file looks like:

- name: Create code coverage
        uses: artiomtr/jest-coverage-report-action@v2.0-rc.5
        with:
          github-token: ${{ secrets.GITHUB_TOKEN }}
          threshold: 11
          coverage-file: ./coverage/report.json
          base-coverage-file: ./coverage/master-report.json
          annotations: coverage
          skip-step: all
          working-directory: ./src/js/

**Comment by ArtiomTr** [comment, marked as answer]
Hello @prabhasjoshi,
Looks like you're using `coverage-file` as well as `base-coverage-file`. By specifying these two options, you're telling the action, that coverage collection logic should be skipped, and read from specified files. Could you please make sure that your `./coverage/report.json` file as well as `./coverage/master-report.json` contain two different reports?
